graphql/server.go

```diff
@@ -157,9 +159,12 @@ func (c *conn) handleSubscribe(id string, subscribe *subscribeMessage) error {
 		ctx = c.makeCtx(ctx)
 
 		start := time.Now()
+		span, ctx := opentracing.StartSpanFromContext(ctx, "thunder.subscription")
+		ext.SamplingPriority.Set(span, 1)
```

Comment:
sgtm. Locally, it looks like this code will execute but the spans will not be reported anywhere when there is no tracer.

Do you know if we can get the query's name (if given) from here?